odl/solvers/functional/default_functionals.py

```diff
@@ -1326,6 +1376,185 @@ def convex_conj(self):
                                  constant=constant)
 
 
+class NuclearNorm(Functional):
+
+    """The nuclear norm.
```

Comment:
Its the nuclear norm ofc, its nuclear and its a norm.

With that said I may add some more info for you infidels who can decipher this! ;)